src/MembershipsSDKAdapter.js

```diff
@@ -0,0 +1,110 @@
+import {BehaviorSubject, throwError} from 'rxjs';
+import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-interfaces';
+
+export default class MembershipsSDKAdapter extends MembershipsAdapter {
+  /**
+   * Gets the active members in a meeting
+   *
+   * @static
+   * @private
+   * @param {object} members Members object from meeting, keyed by ID
+   * @returns {Array}
+   * @memberof MembershipsSDKAdapter
+   */
+  static getActiveMembers(members) {
+    return Object.values(members)
+      .filter((member) => !member.isInMeeting)
+      .map((member) => ({
+        id: member.id,
+      }));
+  }
+
+  constructor(datasource) {
```

Comment:
Can I suggest an order of methods for a class? Maybe I can open a PR later in the contribution guide to make sure it's standardized along all of our classes.
This is how I would suggest the order:
1. Constructor <- Class constructor always goes first
2. Static methods <- Since they're not part of the instance
3. Interface-implementation methods <- Since component system users will be familiar with those the order should match that of the interface
4. Other helper/private methods